### ISSUE TITLE
HOME environment variable incorrect.

### ISSUE THREAD

**Issue by jbearden0** [corbinlc/GNURootDebian#175]
First of all this program is amazing, thanks! I have SSH up and running, and all that works great. However, when I want to use the app to log in on my phone directly, I am logged in as root but the HOME environment variable is set to /home. When I login via SSH it is correct. I can fix it with "export HOME=/root" but is there a way to not have to do that?

**Comment by jbearden0** [comment]
Probably an easier/better way to do what I want is to "su -" instead. Is there a way to do that automatically?

**Comment by pzi123** [comment]
I had the same issue including inability to change permissions on files in /home.  The cause is that this directory is mapped to /sdcard (/storage/emulated/0) which is mounted with default_permissions.
I removed the mapping in /support/execInProot and changed /home to be symbolic link to /root.

**Comment by drhuffman12** [comment]
Wouldn't "changed /home to be symbolic link to /root" make it difficult to add users [e.g.: in case you want something to run as non-root]?

**Comment by cdrasmussen** [comment]
Run "login name" is the best for me but that brings up the problem that the
only file system that normal permissions behavior is in /usr/local. I put
my home directory there.

+CDR

**Comment by pzi123** [comment]
The /usr/local is on the same filesystem as /root - as mapped by the '-r /data/user/0/com.gnuroot.debian/debian' in the proot call in /support/execInProot.
 Using non root users is a very good idea. One way would be to use xdm that starts Xvnc and allows other users to log into X11 session instead of starting tightvncserver as root.

**Comment by cdrasmussen** [comment]
I don't start VNC server as root but login as the non root user before
starting VNC server, just as you would on any Unix.

+CDR

**Comment by ssj-gz** [comment]
Out of interest, where is the HOME=/home initially coming from? From googling, it seems that HOME should be set by "login" based on the contents of /etc/passwd, but my /etc/passwd assigns /root, not /home, as HOME for the root user.

Is it hardcoded/ overridden somewhere in GNURoot?

**Comment by AdamMatthewLTM** [comment]
Hi all,

I am currently assisting the developers in sorting relevant issues for GNURoot Debian on GitHub. If the issue discussed is still relevant, could you please reply to this thread and state the issue needs to be resolved and I will attempt to sort it for you ASAP. If the issue is no longer relevant, please reply to this thread and state it has been resolved, and I will suggest this thread be closed.

Furthermore, if a particular issue is considered to be high priority, please state that you are requesting it to be considered as high priority and I will mark it as such.

Thank you all so much for your time and understanding,
-Adam

**Comment by corbinlc** [comment]
We are putting our efforts into a new, but similar, app called UserLAnd.  It is now in beta and it sets the HOME variable in a normal fashion.  To join the beta of UserLAnd, please go here:  https://play.google.com/apps/testing/tech.ula
